content/integrations/powerdns.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Datadog-PowerDNS Integration
+integration_title: PowerDNS Recursor
+kind: integration
+git_integration_title: powerdns_recursor
+newhlevel: true
+doclevel: basic
+---
+
+# Overview
+
+Connect your PowerDNS Recursor to Datadog to:
+
+* Visualize its performance
+* Understand query latency
+* Get alerted when something fails or when you're under attack.
```

Comment:
One tiny formatting issue: the first two points have no period, but the last one does. I think for consistency, you should remove the period.